components/server/src/projects/projects-service.ts

```diff
@@ -236,12 +234,6 @@ export class ProjectsService {
             await this.auth.removeProjectFromOrg(installer.id, teamId, project.id);
             throw err;
         }
-        await this.scmService.installWebhookForPrebuilds(project, installer);
```

Comment:
How do we educate people that they need to enable Prebuilds now? :thinking:

Reply:
1. After project creation, we render a link to project's settings. 
2. Yes. 

But in general, speaking of educating users: do you think it was commonly known that prebuilds are enabled automatically? 
This is indeed a point we're going to address with updates to design and messages.